radar-upload-frontend/src/axios/file.js

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios';
+import uuidv1 from 'uuid/v1';
+import { downLoadFile } from '@/helpers';
+
+export default {
```

Comment:
I see different `.js` files for subjects and projects, but source-type requests and all record processing logic sit here. Maybe we can rename this file to `record.js` and move out the source-type to source-type.js?

Reply:
Yeah the file name should be changed to record.js. For the source-type, at the moment I find it not necessary to change to another file since it is attached to uploading files. If later we have more reference data I will do it.